src/components/Router.tsx

```diff
@@ -24,6 +24,23 @@ import { WarningPanel } from '@backstage/core';
 export const isPluginApplicableToEntity = (entity: Entity) =>
   entity?.metadata.annotations?.[FIREBASE_FUNCTION_IDS];
 
+export const isOnlyOneFirebaseFunction = (entity: Entity) => {
```

Comment:
I'm not sure this is the right place for these. I understand they're logically similar to isPluginApplicableToEntity but I think we'd like to keep these "private" to this plugin IMO.